test/NuGet.Core.FuncTests/NuGet.Protocol.FuncTest/NuGet.Protocol.FuncTest.csproj

```diff
@@ -31,8 +31,9 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <PostBuildEvent Condition="'$(OS)' == 'Windows_NT'">
-      xcopy /diye $(ArtifactsDirectory)TestablePlugin\$(BuildVariationFolder)\bin\$(Configuration)\$(TargetFramework)\win7-x64\*  $(MSBuildProjectDirectory)\bin\$(Configuration)\$(TargetFramework)\win7-x64\TestablePlugin\
+    <PostBuildEvent Condition="'$(OS)' == 'Windows_NT'">xcopy /diye ^
+    $(ArtifactsDirectory)TestablePlugin\$(BuildVariationFolder)\bin\$(Configuration)\$(TargetFramework)\*    ^
+    $(MSBuildProjectDirectory)\bin\$(Configuration)\$(TargetFramework)\win7-x64\TestablePlugin\
```

Comment:
The line splitting is opinionated (in other words my gut reaction is I prefer the old, long line syntax), I don't see any differences in the arguments being passed, in a file with no functional changes.